tests/Makefile

```
@@ -2,28 +2,59 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
+# =========================
+# Usages
+# - cleanup: make clean
+# - build specific variant of test: make arch=[wormhole|blackhole] test=[TEST_NAME_CPP] variant=[VARIANT]
+# - build default variant of test: make arch=[wormhole|blackhole] test=[TEST_NAME_CPP]
+# =========================
+
+
 # =========================
 # Toolchain and Directories
 # =========================
 CXX_VERSION     := c++17
 TOOL_PATH       ?= sfpi/compiler/bin
 
+
 # =========================
 # Architecture Selection
 # =========================
-ARCH            := $(if $(archname),$(archname),$(CHIP_ARCH))
+ARCH            := $(if $(arch),$(arch),$(CHIP_ARCH))
```

The $(or ...) function returns the first non-empty argument. So if both `arch` and `CHIP_ARCH` are non-empty, it will take `arch`.
```suggestion
ARCH            := $(or $(arch),$(CHIP_ARCH))
```